packages/console/src/pages/ApiResourceDetails/components/DeleteForm/index.tsx

```diff
@@ -56,28 +56,27 @@ const DeleteForm = ({ id, name, onClose }: Props) => {
             title="admin_console.api_resource_details.delete"
             onClick={handleDelete}
           />
-        </div>
+        </>
       }
+      className={styles.content}
       onClose={onClose}
     >
-      <div className={styles.content}>
-        <div className={styles.description}>
-          <Trans
-            t={t}
-            i18nKey="api_resource_details.delete_description"
-            values={{ name }}
-            components={{ span: <span className={styles.hightlight} /> }}
-          />
-        </div>
-        <TextInput
-          value={inputName}
-          placeholder={t('api_resource_details.enter_your_api_resource_name')}
-          hasError={inputMismatched}
-          onChange={(event) => {
-            setInputName(event.currentTarget.value);
-          }}
+      <div className={styles.description}>
+        <Trans
```

Comment:
Wondering why use `Trans` instead of `t(xxx)`, though is a history issue.

Reply:
We will have styles in the part of i18n text.
<img width="591" alt="image" src="https://user-images.githubusercontent.com/10806653/160328036-4ef3632b-55df-44af-a086-a68fc9e919ba.png">

Reply:
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1647597293514959